tests/integration/test_logcollector/test_command_monitoring/test_command_execution.py

```diff
@@ -0,0 +1,214 @@
+    """Get internal configuration."""
+    return local_internal_options
+
+
+def dbg_reading_command(command, alias, log_format):
+    """Check if the (previously known) output of a command ("echo") is displayed correctly.
+
+    It also checks if the "alias" option is working correctly.
+
+    Args:
+        command (str): Command to be monitored.
+        alias (str): An alternate name for the command.
+        log_format (str): Format of the log to be read ("command" or "full_command").
+
+    Raises:
+        TimeoutError: If the command monitoring callback is not generated.
+    """
+    prfx = prefix
+    output = command
+    output = output.replace('echo ', '')
+
+    if log_format == 'full_command':
+        msg = fr"^{output}'"
+        prfx = ''
+    else:
+        msg = fr"DEBUG: Reading command message: 'ossec: output: '{alias}': {output}'"
+
+    wazuh_log_monitor.start(timeout=5, callback=monitoring.make_callback(pattern=msg, prefix=prfx),
+                            error_message=logcollector.GENERIC_CALLBACK_ERROR_COMMAND_MONITORING)
+
+
+def dbg_read_lines(command):
+    """
+    Check if the "DEBUG: Read <number> lines from command <command>" line is displayed correctly.
+
+    Args:
+        command (str): Command to be monitored.
+
+    Raises:
+        TimeoutError: If the command monitoring callback is not generated.
+    """
+    msg = fr"lines from command '{command}'"
+
+    wazuh_log_monitor.start(timeout=60, callback=monitoring.make_callback(pattern=msg, prefix=prefix, escape=True),
+                            error_message=logcollector.GENERIC_CALLBACK_ERROR_COMMAND_MONITORING)
+
+
+def test_command_execution(get_local_internal_options, configure_local_internal_options, get_configuration,
+                           configure_environment, restart_logcollector):
+    """Check if the Wazuh run correctly with the specified command monitoring configuration.
+
+    Ensure command monitoring allow the specified attributes. Also, in the case of manager instance, check if the API
+    answer for localfile block coincides.
+
+    Args:
+        get_local_internal_options (fixture): Get internal configuration.
+        configure_local_internal_options (fixture): Set internal configuration.
+        get_configuration (fixture): Get configurations from the module.
+        configure_environment (fixture): Configure a custom environment for testing.
+        restart_logcollector (fixture): Reset log file and start a new monitor.
+
+    Raises:
+        TimeoutError: If the command monitoring callback is not generated.
+        AssertError: In the case of a server instance, the API response is different that the real configuration.
+    """
+    cfg = get_configuration['metadata']
+
+    log_callback = logcollector.callback_monitoring_command(cfg['log_format'], cfg['command'],
+                                                            prefix=prefix, escape=True)
+    wazuh_log_monitor.start(timeout=5, callback=log_callback,
```

Comment:
You can import the default_timeout from `wazuh_testing` and make this change:
```suggestion
    wazuh_log_monitor.start(timeout=global_parameters.default_timeout, callback=log_callback,
```
By default, this is set to `5`.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1257/commits/1e33fb8c458aeb8a738e22706713ec98fbf25703